java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/obf/preparation/IndexCreator.java

```diff
@@ -514,7 +515,8 @@ public File generateIndexes(File[] readFile, IProgress progress, IOsmStorageFilt
 		this.indexAddressCreator = new IndexAddressCreator(logMapDataWarn, settings);
 		this.indexMapCreator = new IndexVectorMapCreator(logMapDataWarn, mapZooms, renderingTypes, settings, propagateToNodes);
 		this.indexRouteCreator = new IndexRouteCreator(renderingTypes, logMapDataWarn, settings, propagateToNodes);
-		this.indexRouteRelationCreator = new IndexRouteRelationCreator(logMapDataWarn, mapZooms, renderingTypes, settings);
+		this.indexRouteRelationCreatorOld = new IndexRouteRelationCreatorOld(logMapDataWarn, mapZooms, renderingTypes, settings);
```

Comment:
Index route relation old to be deleted? we need to have have just one class at least here